lib/schemaUtils.js

```diff
@@ -198,6 +199,85 @@ module.exports = {
 
   safeSchemaFaker: safeSchemaFaker,
 
+  analyzeSpec: function (spec) {
+    var size,
+      numberOfRefs = 0,
+      specString,
+      numberOfRequests = 0;
+
+    // Stringify and add whitespaces as there would be in a normal file
+    // To get accurate disk size
+    specString = JSON.stringify(spec, null, 2);
+    // Size in MB
+    size = Buffer.byteLength(specString, 'utf8') / 1024 / 1024;
```

Comment:
/ (1024 * 1024)
Generally a good idea to avoid having to rely on operator precedence in any form